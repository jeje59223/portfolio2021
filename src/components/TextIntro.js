import React from "react";
import cv from "../images/cv.pdf";

const TextIntro = () => {
    return (
        <div className="text-intro">
            <div className="preTxt font-italic mb-3">Bonjour, moi c'est Jérôme</div>
            <h1>Développeur Web</h1>
            <a href={cv} className="btn btn-dark mt-5" target="_blank" rel="noreferrer">Télécharger mon CV !</a>
        </div>
    )
}

export default TextIntro;