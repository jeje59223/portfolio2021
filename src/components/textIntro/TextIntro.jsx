import React from 'react';
import cv from '../../images/cv.pdf';
import '../../App.css';

const TextIntro = () => (
  <header id="top">
    <div className="container">
      <div className="text-intro">
        <div className="preTxt font-italic mb-3">Bonjour, moi c'est Jérôme</div>
        <h1>Développeur Web</h1>
        <a href={cv} className="btn btn-dark mt-5" target="_blank" rel="noreferrer">Télécharger mon CV !</a>
      </div>
    </div>
  </header>
);

export default TextIntro;
