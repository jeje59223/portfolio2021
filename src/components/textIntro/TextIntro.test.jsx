import toJson from 'enzyme-to-json';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import TextIntro from "./TextIntro";

configure({ adapter: new Adapter() });

describe('<TextIntro>', () => {
  it('should render TextIntro Component', () => {
    const wrapper = shallow(<TextIntro />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});