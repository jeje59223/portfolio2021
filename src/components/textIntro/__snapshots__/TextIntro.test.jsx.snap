// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextIntro> should render TextIntro Component 1`] = `
<div
  className="text-intro"
>
  <div
    className="preTxt font-italic mb-3"
  >
    Bonjour, moi c'est Jérôme
  </div>
  <h1>
    Développeur Web
  </h1>
  <a
    className="btn btn-dark mt-5"
    href="cv.pdf"
    rel="noreferrer"
    target="_blank"
  >
    Télécharger mon CV !
  </a>
</div>
`;
